teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Teacher < Person
+  # contructor method
+  def initialize(specialization)
+    super(35, 'John Doe')
+    @specialization = specialization
+  end
```

Comment:
- [ ] The constructor extends the parent's constructor by adding @classroom and a parameter for it. Also, when creating the super method, you are supposed to extend the functionalities from the Parent class.  Kindly implement it just as I have suggested to you above.